### PR TITLE
Heading blocks

### PR DESCRIPTION
**Pull request by melissag-ensemble**
## Description

EDS equivalents of the following Gatsby blocks:
- [Heading1](https://github.com/adobe/aio-theme/blob/main/packages/gatsby-theme-aio/src/components/Heading/index.js#L136)
- [Heading2](https://github.com/adobe/aio-theme/blob/main/packages/gatsby-theme-aio/src/components/Heading/index.js#L137)
- [Heading3](https://github.com/adobe/aio-theme/blob/main/packages/gatsby-theme-aio/src/components/Heading/index.js#L138)
- [Heading4](https://github.com/adobe/aio-theme/blob/main/packages/gatsby-theme-aio/src/components/Heading/index.js#L139)
- [Heading5](https://github.com/adobe/aio-theme/blob/main/packages/gatsby-theme-aio/src/components/Heading/index.js#L140)
- [Heading6](https://github.com/adobe/aio-theme/blob/main/packages/gatsby-theme-aio/src/components/Heading/index.js#L141)

## Screenshots (if appropriate):
Untitled

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: hlx_statics/components/divider.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+export default function createDivider(size, style, orientation = 'horizontal') {
```
Comment (melissag-ensemble) [inline hlx_statics/components/divider.js:1]: EDS equivalent of Gatsby [Divider](https://github.com/adobe/aio-theme/blob/main/packages/gatsby-theme-aio/src/components/Divider/index.js#L18) component

File: hlx_statics/components/link.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
+export default function decorateLink(link, variant = 'primary') {
```
Comment (melissag-ensemble) [inline hlx_statics/components/link.js:1]: EDS equivalent of Gatsby [Link](https://github.com/adobe/aio-theme/blob/main/packages/gatsby-theme-aio/src/components/Link/index.js#L18) component

File: hlx_statics/components/heading.js [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+function Anchor({ id }) {
+  const div = document.createElement('div');
+  div.setAttribute('aria-hidden', 'true');
+  div.setAttribute('id', id);
+  div.style.setProperty('position', 'relative');
+  div.style.setProperty('top', 'calc(-1 * var(--spectrum-global-dimension-size-800)');
+  return div;
+}
+
+export default function decorateHeading(level, block) {
```
Comment (melissag-ensemble) [inline hlx_statics/components/heading.js:17]: EDS equivalent of Gatsby [Heading](https://github.com/adobe/aio-theme/blob/main/packages/gatsby-theme-aio/src/components/Heading/index.js#L34) component

File: hlx_statics/components/divider.js [hunk 4]
```diff
@@ -0,0 +1,15 @@
+export default function createDivider({ size, style, orientation = 'horizontal' }) {
```
Comment (melissag-ensemble) [inline hlx_statics/components/divider.js:1]: EDS equivalent of Gatsby [Divider](https://github.com/adobe/aio-theme/blob/main/packages/gatsby-theme-aio/src/components/Divider/index.js#L18) component

File: hlx_statics/components/link.js [hunk 5]
```diff
@@ -0,0 +1,6 @@
+export default function decorateLink({ link, variant = 'primary' }) {
```
Comment (melissag-ensemble) [inline hlx_statics/components/link.js:1]: EDS equivalent of Gatsby [Link](https://github.com/adobe/aio-theme/blob/main/packages/gatsby-theme-aio/src/components/Link/index.js#L18) component

File: hlx_statics/components/heading.js [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+function Anchor({ id }) {
+  const div = document.createElement('div');
+  div.setAttribute('aria-hidden', 'true');
+  div.setAttribute('id', id);
+  div.style.setProperty('position', 'relative');
+  div.style.setProperty('top', 'calc(-1 * var(--spectrum-global-dimension-size-800)');
+  return div;
+}
+
+export default function decorateHeading({ level, block }) {
```
Comment (melissag-ensemble) [inline hlx_statics/components/heading.js:17]: EDS equivalent of Gatsby [Heading](https://github.com/adobe/aio-theme/blob/main/packages/gatsby-theme-aio/src/components/Heading/index.js#L34) component

**Review by timkim — APPROVED** [review]

**Pull request merged**